### PR TITLE
CI: optional compilation with line editing

### PR DESCRIPTION
**Pull request by aehyvari**
Tiny PR for allowing the user to specify as an environment variable to compile opensmt with line editing.  Useful for people who want to quickly test corner case behaviour on opensmt when running locally ci scripts.

### CODE REVIEW

File: ci/run_travis_commands.sh [hunk 1]
```diff
@@ -11,6 +11,7 @@ fi
 
 cmake -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
       -DCMAKE_CXX_FLAGS="${FLAGS}" \
+      -DENABLE_LINE_EDITING="${ENABLE_LINE_EDITING}" \
```
Comment (blishko) [inline ci/run_travis_commands.sh:14]: Isn't this doing the same as the `ENABLE_LINE_EDITING` two lines below?

Comment (aehyvari) [reply ci/run_travis_commands.sh:14]: Yes, looks like that :)

I guess we won't need this one!
